resources/js/Pages/Playground.vue

```diff
@@ -1,27 +1,37 @@
 <template>
-  <div>
+  <div lang="en">
     <Head title="Mismatch Finder - Component Playground" />
-    <h1>Mismatch Finder - Playground!</h1>
+    <h2>Mismatch Finder - Playground!</h2>
     <p>Feel free to throw any component you want to try, in here</p>
-    <auth-widget v-bind:user="user" />
-    <text-area :rows="10" label="I am from Wikit" resize="horizontal" value="With a default value" @input="printVal" />
-    <wikit-button type="progressive" variant="primary" @click.native="console">Test Wikit Component</wikit-button>
+
+    <wikit-table id="some-table" linearize="desktop">
+        <thead>
+            <tr>
+                <th scope="col">Name</th>
+                <th scope="col">Calories</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr v-for="dessert in desserts" :key="dessert.name">
+                <!-- When using the table make sure to include the
+                    data-header for linearization and accesibility -->
+                <td data-header="Name">{{dessert.name}}</td>
```

Comment:
it is not immediately obvious to me if we can pass anything other than text into this component. wouldn't it make sense to have an example in the playground that introduces some other element in a cell? we will need this right away in the implementation for MF. What do you think?

Reply:
I added an example to the Playground, that illustrates how to use the Link component with a url from the data section.

Reply:
Great. Thanks for adding the example.